tutorials/webpack/alias.md

```diff
@@ -0,0 +1,63 @@
+Aliasing is webpack's handy way to shave time and keystrokes
+off importing frequently used modules. You will need the
+`path` npm module as it is how you will tell webpack where
+to look for those specific files. Using the resolve object
+as the parent object and the alias property as the child object,
```

Comment:
I'd just say "Using the `resolve.alias` property, you can define aliases for frequently imported modules"